packages/ckeditor5-dev-env/lib/release-tools/changelog/writer-options.js

```diff
@@ -47,6 +48,16 @@ module.exports = {
 // - filters out the commit if it should not be visible in the changelog,
 // - makes links to issues and user's profiles on GitHub.
 function transformCommit( commit ) {
+	if ( commit.header.startsWith( 'Merge pull request' ) ) {
```

Comment:
What about merges done from the console? The message is just "Merge ...", so I'll change this check to `startsWith( 'Merge' )`